pegasus/sites.v3/code.org/public/student/middle-high.haml

```diff
@@ -0,0 +1,34 @@
+---
+title: Computer Science Curriculum for Grades 6-12
+theme: responsive
+---
+
+%link{type: "text/css", href: "/css/learn-carousel.css", rel: "stylesheet"}
+%script{type: "text/javascript", src: "/js/bootstrap.min.js"}
```

Comment:
Oops, copied from /learn/legacy but I think it was only used for the top Code Studio carousel. I'll remove.